src/components/reusable/breadcrumbs/breadcrumbs.stories.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright Kyndryl, Inc. 2023
+ */
+
+// External library imports
+import { html } from 'lit';
+import type { Meta, StoryObj } from '@storybook/web-components';
+
+import '@kyndryl-design-system/foundation/components/link';
+
+// Relative component imports
+import './breadcrumbs';
+import './index';
+
+/**
+ * ## Usage
+ *
+ * Here's how you can use the `kyn-breadcrumbs` component with sample items:
+ *
+ * ```javascript
+ *
+ * html`
+ *  <kyn-breadcrumbs>
+ *    <kd-link class="breadcrumb-item" .href="${'/'}">Bridge</kd-link>
+ *    <kd-link class="breadcrumb-item" .href="${'/catalog'}">Catalog</kd-link>
+ *    <span class="breadcrumb-item breadcrumb-last">FinOps</span>
+ *  </kyn-breadcrumbs>
+ * `;
+ * ```
+ *
+ * Remember to import and define the component before using it.
+ */
+
+const meta: Meta = {
+  title: 'Components/Breadcrumbs',
+  component: 'kyn-breadcrumbs',
+  parameters: {
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/6AovH7Iay9Y7BkpoL5975s/Component-Library-for-Dev?node-id=459%3A39033&mode=dev',
+    },
+  },
+};
+
+export default meta;
+
+type Story = StoryObj;
+
+export const Breadcrumbs: Story = {
+  render: () => {
+    return html`<kyn-breadcrumbs>
+      <kd-link class="breadcrumb-item" .href="${'/'}">Bridge</kd-link>
+      <kd-link class="breadcrumb-item" .href="${'/catalog'}">Catalog</kd-link>
+      <span class="breadcrumb-item breadcrumb-last">FinOps</span>
```

Comment:
Should be able to style these elements without needing to add a class by targeting the tag name instead. I would not put the onus on app devs to add these specific classes.

Reply:
You're right, they could technically put anything in the slot and there's not much we can do about it. We just have to provide the guidelines and examples, but asking them to add specific classes is unnecessary and confusing,

Reply:
Hmm we're trying to give them much flexibility with the slot approach but then restrict them with certain tags. It feels strange/hard-coded, especially if later on we have our own components such as <kyn-link> or <Typography> that consuming apps want to put in the slot.

Reply:
That's just how subcomponents work with slots in a web component ecosystem.